Ginger/GingerCoreNET/Reports/RunSetReport.cs

```diff
@@ -261,6 +263,7 @@ public Amdocs.Ginger.CoreNET.Execution.eRunStatus RunSetExecutionStatus
         public void SetLiteDBData(LiteDbRunSet runSet)
         {
             GUID = runSet._id.ToString();
+            RunSetGuid = runSet.GUID;
```

Comment:
also add it for Text file logger and verify if it works

Reply:
verified and made necessary changes